projects/igniteui-angular/src/lib/snackbar/snackbar.component.ts

```diff
@@ -102,6 +104,22 @@ export class IgxSnackbarComponent extends IgxNotificationsDirective
      */
     @Output() public animationDone = new EventEmitter<ToggleViewEventArgs>();
 
+    /**
+     * @hidden
+     */
+    public get buttonTemplate(): boolean {
```

Comment:
I don't think you need this. Did it now work by simply putting an identifier on the ng-content and then checking that that element exists?